inscriptions/ISic0963.xml

```diff
@@ -174,94 +179,18 @@
-                    </expan>
-                    καʹ.
-                    <lb n="8"/>
-                </ab>
-            </div>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI175470">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="VM">
                 <ab>
-                    <lb n="1"/>
-                    <expan>
-                        <abbr>Ἄλφ</abbr>
-                        <ex>ις</ex>
-                    </expan>
-                    Κλῶδις ἐτελεύτησεν πρὸ η
-                    <expan>
-                        <abbr>Εἰδ</abbr>
-                        <ex>ῶν</ex>
-                    </expan>
-                    #⁵⁶
-                    <certainty match="preceding-sibling::node()" locus="value"/>
-                    <lb n="2"/>
-                    <expan>
-                        <abbr>Νοβεμ</abbr>
-                        <ex>βρίων</ex>
-                    </expan>
-                    <expan>
-                        <abbr>ἡμέρ</abbr>
-                        <ex>ᾳ</ex>
-                    </expan>
-                    Ἡλίου, υἱὸς Λουκίου καὶ<lb n="3"/>Καικειλίας τῶν κειμένων ἰς τὸν πυλῶνα<lb n="4"/>ἰσερχομένων ἰς δεξιά, ἄρξας τὴν δευτέ-<lb n="5"/>ραν ἀρχὴν καὶ πρεσβίας πρεσβεύσας
-                    πρὸς<lb n="6"/>βασιλέα καὶ γ παραπονπε
-                    <supplied reason="omitted">ύ
-                    </supplied>
-                    <expan>
-                        <abbr>ς</abbr>
-                        <ex>?</ex>
-                    </expan>
-                    .
-                    <lb n="7"/>
-                    <expan>
-                        <abbr>ἔζησ</abbr>
-                        <ex>εν</ex>
-                    </expan>
-                    ἔτη λβ μῆνες ι
-                    <expan>
-                        <abbr>ἡμέ</abbr>
-                        <ex>ρας</ex>
-                    </expan>
-                    καʹ.
-                    <lb n="8"/>
+                    <lb n="1"/><expan><abbr>Ἄλφ</abbr><ex>ιος</ex></expan> Κλῶδις ἐτελεύτησεν πρὸ η <expan><abbr>εἰδ</abbr><ex>ῶν</ex></expan>
+                    <lb n="2"/><expan><abbr>Νοβεμ</abbr><ex>βρίων</ex></expan> <expan><abbr>ἡμέρ</abbr><ex>ᾳ</ex></expan> Ἡλίου υἱὸς Λουκίου καὶ
+                    <lb n="3"/>Καικει<supplied reason="omitted">λί</supplied>ας τῶν κειμένων ἰς τὸν πυ<supplied reason="omitted">λῶ</supplied><supplied reason="lost">να</supplied>
+                    <lb n="4"/>ἰσερχομένων ἰς δεξιά ἄρξας τὴν δευτέ
+                    <lb n="5" break="no"/>ραν ἀρχὴν καὶ πρε<supplied reason="omitted">σ</supplied>βίας πρεσβεύ<supplied reason="omitted">σ</supplied>ας πρὸς
+                    <lb n="6"/>βασιλέα καὶ γ παραπονπε&#x0342;ς·
+                    <lb n="7"/><expan><abbr>ἔζησ</abbr><ex>εν</ex></expan> ἔτη <hi rend="supraline"><num value="31">λα</num></hi> μῆνες ι <expan><abbr>ἡμέ</abbr><ex>ραι</ex></expan> <hi rend="supraline"><num value="21">κα</num></hi>
```

Comment:
Encode the ι for the month number.

Reply:
there is also the eta (first line I think) for the month. so should I put them inside the number element too?

Reply:
Yes. I'll put a note to discuss a bit more about which dates we want to encode fully.

Reply:
all numbers should be encoded in this basic num@value form I think. The question of date encoding is perhaps a different issue. Worth noting that Laura is fundamentally interested in theq uestion of encoding ages at death / span lived.